### ISSUE TITLE
Your scanner is missing a heuristic exception...

### ISSUE THREAD

**Issue by kevindawson** [kevindawson/App-Midgen#6]
Your scanner is missing a heuristic exception, that when it does see a

``` perl
require MRO::Compat;
```

It should automatically **discard** the 

``` perl
require mro
```

It found, and subsequently the perl-minversion 5.10 req that comes with it.

**Comment by kevindawson** [comment]
``` perl
Perl::MinimumVersion::Reason  {
    public methods (5) : element, explanation, new, rule, version
    private methods (0)
    internals: {
        element   PPI::Statement::Include,
        rule      "_perl_5010_pragmas",
        version   version
    }
}
```

**Comment by kevindawson** [comment]
``` perl
Perl::MinimumVersion::Fast  {
    public methods (5) : minimum_explicit_version, minimum_syntax_version, minimum_version, new, version_markers
    private methods (2) : _build_minimum_explicit_version, _build_minimum_syntax_version
    internals: {
        minimum_explicit_version   undef,
        minimum_syntax_version     version,
        version_markers            {
            5.010   [
                [0] "use mro"
            ]
        }
    }
}
```

**Comment by kevindawson** [comment]
I will e-mail the autors of

``` perl
Perl::MinimumVersion
Perl::MinimumVersion::Fast
```

to see if this belongs in there world

**Comment by kevindawson** [comment]
the above issue is from the following code snippet

``` perl
BEGIN {
  # use Module::Runtime to work around the 5.8 require bugs
  if ($] < 5.009_005) {
    Module::Runtime::require_module('MRO::Compat');
  }
  else {
    require mro;
  }
}
```

**Comment by kevindawson** [comment]
fixed in 0.31_05

for more info see: [mro - MRO::Compat catch](http://kevindawson.github.io/bowtie/2014/03/22/app-midgen-010/)
